nav2_costmap_2d/plugins/voxel_layer.cpp

```diff
@@ -108,11 +110,12 @@ void VoxelLayer::matchSize()
 
 void VoxelLayer::reset()
 {
+  voxel_pub_->on_deactivate();
   deactivate();
   resetMaps();
-  voxel_grid_.reset();
```

Comment:
Why was this removed?

Reply:
@SteveMacenski 
While executing the `reset()`, the `resetMaps()` calls `voxel_grid_.reset()`, so I thought it was redundant.

```
void VoxelLayer::reset()
{
  deactivate();
  resetMaps();
  voxel_grid_.reset();
  activate();
  undeclareAllParameters();
}

void VoxelLayer::resetMaps()
{
  Costmap2D::resetMaps();
  voxel_grid_.reset();
}
```

Reply:
Looking at the git history in nav1 that appears to be the case